server/src/api/controllers/commandController.js

```diff
@@ -7,7 +7,6 @@ import CommandHandler from '../../game/commandHandler.js'
 export async function newCommand(req, res) {
   console.log('new command:', req.body)
   const { token, command, pwd, output, error, ...additionalData } = req.body
-  console.log('additionalData:', additionalData)
```

Comment:
So can we still see additional data from log after this? Or it is just move to another place?

Reply:
Oh, I just thought we don't need to see that again. You can see the whole body from req.body after all